mesonbuild/backend/ninjabackend.py

```diff
@@ -2360,7 +2360,7 @@ def generate_custom_target_clean(self, outfile, trees):
         e.add_item('COMMAND', [sys.executable,
                                self.environment.get_build_command(),
                                '--internal', 'cleantrees', d_file])
-        e.add_item('description', 'Cleaning CustomTarget directories')
+        e.add_item('description', 'Cleaning custom targets directories.')
```

Comment:
Are you sure? The rule is meant to clean a set of directories that belong to a set of targets, so shouldn't they both be plural?